src/github-patcher.css

```
@@ -11,15 +11,11 @@ a[href$='/forks'] strong,
 /* Third party stats. "GitHub Readme Stats" */
 /* NOTE: Do not hide other endpoints like `api/top-langs` */
 a:has(> img[data-canonical-src^='https://github-readme-stats.vercel.app/api?']) {
-  display: none;
+  display: none !important;
```

Don't change non related styles.

```suggestion
  display: none;
```